benchmarks/benchmark_serving.py

```diff
@@ -611,7 +611,11 @@ def main(args: argparse.Namespace):
         args.request_rate if args.request_rate < float("inf") else "inf"
     )
 
-    metrics_json = {**metrics_json, **benchmark_result}
+    metrics_json = {
+        **metrics_json,
+        **benchmark_result,
+        **json.loads(args.additional_metadata_metrics_to_save),
```

Comment:
Should this metadata be appended to dimensions_json instead of metrics_json? Since metrics values are numerical, mainly for benchmark results.

Reply:
The metadata name is misleading. The values I've been adding are numerical right now. I could rename it from "metadata" to something else?

Reply:
These `ici_fsdp_parallelism, ici_autoregressive_parallelism, ici_tensor_parallelism` are you referring to the maxtext config? These are metadata but not metrics?

Reply:
Shared with you an explanation of the metadata via chat.

Reply:
Per our chats, will stringify the ici values, and pass it into the `dimensions_json`.